src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,293 @@
+            readCalls(otherWorkflowIdTextboxValue)
+          }
+        }, ['Continue'])
+      ]),
+      metadataFetchError && [
+        'Error loading workflow metadata',
+        h(ErrorView, { error: metadataFetchError })
+      ]
+    ])
+  }
+
+  const step2 = () => {
+    const selectedCallIndex = Utils.cond(
+      [otherIndexDropdownValue !== undefined, () => otherIndexDropdownValue],
+      [otherWorkflowMetadata && otherCallFqnDropdownValue && otherWorkflowMetadata.calls[otherCallFqnDropdownValue].length === 1,
+        () => otherWorkflowMetadata.calls[otherCallFqnDropdownValue][0].shardIndex],
+      () => undefined
+    )
+
+    const otherCallSucceeded = otherWorkflowMetadata && otherCallFqnDropdownValue && selectedCallIndex !== undefined &&
+      _.some({ shardIndex: selectedCallIndex, executionStatus: 'Done' }, otherWorkflowMetadata.calls[otherCallFqnDropdownValue])
+
+    return h(Fragment, [
+      div({ style: { display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected workflow B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.codeFont } }, otherWorkflowId)
+        ]),
+        (otherWorkflowMetadata || metadataFetchError) && resetLink(() => resetWorkflowSelection(''))
+      ]),
+      div({ style: { paddingTop: '0.5rem', fontSize: 16, fontWeight: 500 } }, ['Step 2: Select which call in that workflow you expected to cache from']),
+      otherWorkflowMetadata ?
+        div([
+          div({ style: { marginTop: '1rem', display: 'flex', flexDirection: 'row', alignItems: 'center' } }, [
+            h(IdContainer, [id => h(Fragment, [
+              label({ htmlFor: id, style: { paddingRight: '0.5rem' } }, ['Call name:']),
+              div({ style: { flex: '1' } }, [
+                h(Select, {
+                  id,
+                  isSearchable: false,
+                  options: otherCallFqnSelectionOptions(otherWorkflowMetadata.calls),
+                  value: otherCallFqnDropdownValue,
+                  onChange: v => {
+                    setOtherIndexDropdownValue(undefined)
+                    setOtherCallFqnDropdownValue(v.value)
+                  }
+                })
+              ])
+            ])])
+          ]),
+          otherCallFqnDropdownValue && div({ style: { marginTop: '0.25rem', marginBottom: '0.5rem', display: 'flex', flexDirection: 'row', alignItems: 'center' } }, [
+            h(IdContainer, [id => h(Fragment, [
+              label({ htmlFor: id, style: { paddingRight: '0.5rem' } }, ['Shard Index:']),
+              div({ style: { flex: '1' } }, [Utils.cond(
+                [selectedCallIndex === -1, () => 'N/A (not scattered)'],
+                [_.size(otherWorkflowMetadata.calls[otherCallFqnDropdownValue]) === 1, `${otherWorkflowMetadata.calls[otherCallFqnDropdownValue][0].shardIndex} (exactly one shard in scatter)`],
```

Comment:
The 'value' items in each pair need to be functions. Technically `cond` will accept bare values, but it's deprecated and will be removed soon.
```suggestion
                [_.size(otherWorkflowMetadata.calls[otherCallFqnDropdownValue]) === 1, () => `${otherWorkflowMetadata.calls[otherCallFqnDropdownValue][0].shardIndex} (exactly one shard in scatter)`],
```